docs/api/usage.md

```diff
@@ -51,8 +51,7 @@ array(
 )
 ```
 
-!!! note
-    A few specialized actions like `getsingle` or `getvalue` may return success in a different format.
+*Note*: A few specialized actions like `getsingle` or `getvalue` may return success in a different format.
```

Comment:
I think the `!!! note` seems OK?

![Capture d’écran de 2019-05-09 15-13-23](https://user-images.githubusercontent.com/254741/57480213-158bdc80-726d-11e9-8f06-a429eefe186c.png)